scripts/README.md

```diff
@@ -71,3 +71,37 @@ EVC_IDS='d33539656d8b40,095e1d6f43c745' priority python3 scripts/002_unset_spf_a
 ```
 
 - After that, `kytosd` should be restarted just so `mef_eline` EVCs can get fully reloaded in memory with the expected primary and secondary constraints, this would be the safest route.
+
+### Change ``tag_type`` from integer to string type
+
+[`vlan_type_string.py`](./vlan_type_string.py) is a script to change every ``tag_type`` instance from integer to string type. These istances are found in evcs collection from MongoDB.
+
+```
+    VLAN = 1 to 'vlan'
+    VLAN_QINQ = 2 to 'vlan_qinq'
+    MPLS = 3 to 'mpls'
+```
+
+#### Pre-requisites
+
+- There's no additional Python libraries dependencies required, other than installing the existing `topology`'s, or if you're running in development locally then installing `requirements/dev.in`
```

Comment:
> other than installing the existing `topology`'s

If you could remove this extra phrase, it's OK to reuse the text from topology, but this current phrase needs to either be removed or updated.

Reply:
Removed phrase as well as the next one.